### PR TITLE
Fix startup script name in Dockerfile

### PR DESCRIPTION
**Pull request by hyangtack**
Modifications:

- Change startup script name
- Add ‘latest’ tag to docker image
- Add how to run docker image

### CODE REVIEW

File: site/src/sphinx/setup-installation.rst [hunk 1]
```diff
@@ -60,3 +60,12 @@ To stop the server, use the ``bin/shutdown`` script:
     You can also tail the log file::
 
         $ ./gradlew tail
+
+Run on docker
+--------------------
```
Comment (trustin) [inline site/src/sphinx/setup-installation.rst]: - Run -> Running
- docker -> Docker
- Mismatching number of hyphens - should be of same length with the section title

File: site/src/sphinx/setup-installation.rst [hunk 2]
```diff
@@ -60,3 +60,12 @@ To stop the server, use the ``bin/shutdown`` script:
...
+You can also pull Central Dogma image from `Docker Hub <https://hub.docker.com/r/line/centraldogma>`
```
Comment (trustin) [inline site/src/sphinx/setup-installation.rst]: Missing trailing underscore after the closing backquote?

File: site/src/sphinx/setup-installation.rst [hunk 3]
```diff
@@ -60,3 +60,12 @@ To stop the server, use the ``bin/shutdown`` script:
...
+and then run it on your docker:
+
+.. parsed-literal::
```
Comment (trustin) [inline site/src/sphinx/setup-installation.rst]: - can omit .. parsed literal:: if you end the last sentence with :: i.e. on your Docker::
- docker -> Docker

File: dist/build.gradle [hunk 4]
```diff
@@ -111,7 +111,9 @@ task docker(group: 'Build',
             dependsOn: tasks.distDir,
             type: DockerBuildImage) {
     inputDir = file('.')
-    tag = "line/centraldogma:${project.version}"
+    def name = 'line/centraldogma:'
+    tags = [name + "${project.version}",
+            name + "latest"]
```
Comment (trustin) [inline dist/build.gradle]: - I'm curious how Docker handles snapshot versions. Should we replace 'SNAPSHOT' with a timestamp?
- How do we prevent users from fetching unreleased snapshot versions?

File: site/src/sphinx/client-cli.rst [hunk 5]
```diff
@@ -9,8 +9,8 @@ so the client is available regardless of your current working directory.
 .. note::
 
     In this tutorial, we assume your Central Dogma server is running at ``localhost:36462``. Unless specified,
-    ``dogma`` will connect to ``localhost:36462`` by default. Use the ``--host`` option to connect to other
-    Central Dogma server. e.g. ``--host dogma.example.com:36462``
+    ``dogma`` will connect to ``localhost:36462`` by default. Use the ``--connect`` option to connect to other
```
Comment (minwoox) [inline site/src/sphinx/client-cli.rst]: Thank you~!!

**Pull request merged**